django/api/models/__init__.py

```diff
@@ -9,7 +9,7 @@
 from . import vehicle_zev_type
 from . import vin_decoded_information
 from . import ldv_rebates
-from . import speciality_use_vehicle_incentives
+from . import go_electric_rebates
```

Comment:
I think this is what we were talking about in standup this morning, about renaming models? I see in this PR that the SUVI model was deleted, but there's no new GER model, and there's no migrations associated with the change; if there's issues with creating the migration when a model is renamed and has changed fields, then like Emily said, perhaps try doing 2 migrations (alter fields, make migrations + migrate, then rename, then make migrations + migrate again). Please let me know if you need any help!

Reply:
I think the migrations just weren't staged for some reason so I missed them on the commit. I am adding them in now; but I'm 100% sure it is doing a deletion instead of a rename. When testing the same process again it asked if I renamed a model so I am not sure why this migration didn't check for that.